src/test/java/DukeTest.java

```diff
@@ -0,0 +1,34 @@
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DukeTest {
+    @Test
+    public void parseTodoWorks(){
+        String todo = "study ";
+        String[] test = new String[]{"todo", "study"};
+        Todo task = new Todo(todo,false);
+        Todo task2 = Parser.parseTodo(test);
+        assertEquals(task.toString(),task2.toString(),"Wrong format");
```

Comment:
I think that white spaces can be added here for readability 
```suggestion
        assertEquals(task.toString(), task2.toString(), "Wrong format");
```